charts/acp/values.yaml

```diff
@@ -592,17 +592,14 @@ workers:
   ##
   # podDisruptionBudget: {}
 
-## Fission chart configuration
+## Available FaaS providers: fission, docker, hybrid
```

Comment:
Please add more context here and explain the differences between these modes.

Reply:
You're pointing it to the doc which uses fission: enabled https://cloudentity.com/developers/deployment-and-operations/configure/configure-fission-for-faas/#fission-integration-for-faas
In this PR you're removing this option. I believe this doc should be updated according to this changes